docs/run_results.json

```diff
@@ -0,0 +1 @@
w_web.not_null_snowplow_web_users_last_page_url"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2021-07-07T13:18:48.171935Z", "completed_at": "2021-07-07T13:18:48.178535Z"}, {"name": "execute", "started_at": "2021-07-07T13:18:48.178697Z", "completed_at": "2021-07-07T13:18:48.178705Z"}], "thread_id": "Thread-1", "execution_time": 0.007147789001464844, "message": null, "adapter_response": {}, "unique_id": "test.snowplow_web.not_null_snowplow_web_users_network_userid"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2021-07-07T13:18:48.179266Z", "completed_at": "2021-07-07T13:18:48.184918Z"}, {"name": "execute", "started_at": "2021-07-07T13:18:48.185069Z", "completed_at": "2021-07-07T13:18:48.185076Z"}], "thread_id": "Thread-1", "execution_time": 0.006150960922241211, "message": null, "adapter_response": {}, "unique_id": "test.snowplow_web.not_null_snowplow_web_users_page_views"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2021-07-07T13:18:48.185643Z", "completed_at": "2021-07-07T13:18:48.191344Z"}, {"name": "execute", "started_at": "2021-07-07T13:18:48.191496Z", "completed_at": "2021-07-07T13:18:48.191504Z"}], "thread_id": "Thread-1", "execution_time": 0.006225109100341797, "message": null, "adapter_response": {}, "unique_id": "test.snowplow_web.not_null_snowplow_web_users_sessions"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2021-07-07T13:18:48.192074Z", "completed_at": "2021-07-07T13:18:48.197444Z"}, {"name": "execute", "started_at": "2021-07-07T13:18:48.197586Z", "completed_at": "2021-07-07T13:18:48.197593Z"}], "thread_id": "Thread-1", "execution_time": 0.005875825881958008, "message": null, "adapter_response": {}, "unique_id": "test.snowplow_web.not_null_snowplow_web_users_start_tstamp"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2021-07-07T13:18:48.198111Z", "completed_at": "2021-07-07T13:18:48.204085Z"}, {"name": "execute", "started_at": "2021-07-07T13:18:48.204241Z", "completed_at": "2021-07-07T13:18:48.204248Z"}], "thread_id": "Thread-1", "execution_time": 0.0064640045166015625, "message": null, "adapter_response": {}, "unique_id": "test.snowplow_web.unique_snowplow_web_users_domain_userid"}, {"status": "success", "timing": [{"name": "compile", "started_at": "2021-07-07T13:18:48.204762Z", "completed_at": "2021-07-07T13:18:49.059867Z"}, {"name": "execute", "started_at": "2021-07-07T13:18:49.060400Z", "completed_at": "2021-07-07T13:18:49.060419Z"}], "thread_id": "Thread-1", "execution_time": 0.925055980682373, "message": null, "adapter_response": {}, "unique_id": "test.snowplow_web_integration_tests.dbt_utils_equality_snowplow_web_users_actual_ref_snowplow_web_users_expected_it_"}], "elapsed_time": 13.01059103012085, "args": {"log_format": "default", "write_json": true, "profiles_dir": "/Users/williamwarner/.dbt", "use_cache": true, "compile": true, "version_check": true, "which": "generate", "rpc_method": "docs.generate"}}
```

Comment:
What is this file? Should it be included in a package's source code?

Reply:
I believe it is required to build the doc site.

https://docs.getdbt.com/docs/guides/building-packages#5-deploy-the-docs-for-your-package